packages/client/ui/react-ui/src/testUtils.ts

```diff
@@ -0,0 +1,5 @@
+export const MOCK_API_KEY = "sk_development_12341234";
+export const waitForSettledState = async (callback: () => void) => {
```

Comment:
This feels really naive to me, but I didn't see anything better in the docs and claude couldn't think of anything better either.

Reply:
We'll look later if necessary to make testing the status more robust, but good for now I think